### PR TITLE
Bugfix: fix cdash issue

### PR DESCRIPTION
**Pull request by SusanSun17**
## Description

AWS Codebuild fails saying "Build Failed. No CDASH output"
So I'm attempting to put a fix into ufo-data which has already been applied to ufo, oops, ioda, etc.

### Issue(s) addressed

CDASH issue.

## Acceptance Criteria (Definition of Done)

ufo-data PRs stop failing build.

## Dependencies

None.

## Impact

None.

### CODE REVIEW

**Comment by SusanSun17** [comment]
Sorry @mer-a-o @srherbener - I don't know how you fixed this issue in the other repo's - can I just flag this up and leave it to you the experts!

**Comment by climbfuji** [comment]
> Sorry @mer-a-o @srherbener - I don't know how you fixed this issue in the other repo's - can I just flag this up and leave it to you the experts!

We can take care of this, it is due to the container updates that were made earlier this week.

**Comment by climbfuji** [comment]
@SusanSun17 done, ready to merge

File: CI/buildspec_clang.yml [hunk 1]
```diff
@@ -47,17 +47,11 @@ phases:
...
+      - mkdir /build_container
 
-
-      # Upload branch name and commit sha as CodeBuild artifact to S3
-      - mkdir -p /jcsda/artifacts
-      - echo ${CODEBUILD_GIT_BRANCH} > /jcsda/artifacts/branch_name.txt
-      - echo ${CODEBUILD_RESOLVED_SOURCE_VERSION} > /jcsda/artifacts/commit_sha.txt
-
+      - git lfs install # creates .gitconfig
+      #- cp ~/.gitconfig .
```
Comment (mer-a-o) [inline CI/buildspec_clang.yml]:
```suggestion
```

File: CI/buildspec_clang.yml [hunk 2]
```diff
@@ -98,13 +95,15 @@ phases:
...
       - echo Executing build phase
-      - echo $CODEBUILD_BUILD_SUCCEEDING
       - export BUILD_STATUS="0"
       - echo $BUILD_STATUS
+      - echo $CODEBUILD_BUILD_SUCCEEDING
 
       # configure and build
       - cd /build_container
-      - ecbuild -Wno-dev -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCDASH_OVERRIDE_GIT_BRANCH=$CODEBUILD_GIT_BRANCH -DCTEST_UPDATE_VERSION_ONLY=FALSE -DBUILD_IODA_CONVERTERS=ON /jcsda/ioda-bundle
+      - ls /etc/profile.d/
```
Comment (mer-a-o) [inline CI/buildspec_clang.yml]:
```suggestion
```

**Review by mer-a-o — COMMENTED** [review]
Looks good. 2 small suggestions.

**Review by mer-a-o — APPROVED** [review]

**Comment by climbfuji** [comment]
@danholdaway @mathomp4 This PR is ready to review and merge, all tests passed.

**Pull request merged**